### PR TITLE
[Doris Manager][Doc]Basic User Documents for Doris Manager

### PR DESCRIPTION
**Pull request by JusticeGong**
# Proposed changes

Issue Number: #7986 #6491

## Problem Summary:

Describe the overview of changes.

This PR contains basic usage instruction documents of Doris Manager, including compiling, initializing the platform, space & cluster management and system settings.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

### CODE REVIEW

File: docs/zh-CN/extending-doris/doris-manager/initializing.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+Doris 0.14.0 之后的依赖版本如下：
+
+    GCC 10+, Oracle JDK 1.8+, Python 2.7+, Apache Maven 3.5+, CMake 3.19.2+ Bison 3.0+
+
+
+
+###### 2 预编译部署
+
+使用预编译版本进行安装
```
Comment (luzhijing) [inline docs/zh-CN/extending-doris/doris-manager/initializing.md:115]: 删除本段预编译版本的介绍

**Review by luzhijing — COMMENTED** [review]
1.Remove the part of Pre-complied version

2. Leave a space before and after English letters

3.add the English documents

**Comment by hf200012** [comment]
Add English documents, this picture is a bit big

**Review by hf200012 — APPROVED** [review]
LGTM

**Comment by hf200012** [comment]
English document follow-up supplement

**Pull request merged**